src/tests/user.rs

```diff
@@ -23,37 +27,35 @@ pub struct UserShowPrivateResponse {
     pub user: EncodablePrivateUser,
 }
 
+#[derive(Deserialize)]
+struct CrateList {
```

Comment:
I'm pretty sure these tests can use the `CrateList` defined in tests/all.rs, I'll try after finishing this review.